Lib/test/test_inspect.py

```diff
@@ -1077,6 +1077,51 @@ class C(metaclass=M):
         attrs = [a[0] for a in inspect.getmembers(C)]
         self.assertNotIn('missing', attrs)
 
+class TestIsDataDescriptor(unittest.TestCase):
```

Comment:
These tests are not ran. Add the test class in `run_unittest()` list below.

Reply:
Added